discovery/discovery.go

```diff
@@ -148,12 +148,12 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 // DiscoverEndpoints will make HTTPS requests to find the ac-discovery meta
```

Comment:
@sgotti Sorry for the silence, I have been mulling on this for a few days as I am not really happy with the current relationship between simple/meta. What do you think of the discussion in #140 ?